pragatipant/listify
# Added UI  for add and delete.

Test plan:
- Check out branch addToDoList
- Pull latest code
- Do npm install
- Do npm start
- Verify app loads locally
- Add a task to the list
- Add another task from the list
- Delete a task from the list
